app/src/main/java/com/teambeme/beme/mypage/viewmodel/MyPageViewModel.kt

```diff
@@ -62,7 +76,93 @@ class MyPageViewModel(private val myPageRepository: MyPageRepository) : ViewMode
         })
     }
 
-    fun setDummyWrite() {
+    fun getMyAnswer() {
+        myPageRepository.getMyAnswer(
+            "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaWF0IjoxNjEwMDk5MjQwLCJleHAiOjE2MzYwMTkyNDAsImlzcyI6ImJlbWUifQ.JeYfzJsg-kdatqhIOqfJ4oXUvUdsiLUaGHwLl1mJRvQ",
+            null,
+            null,
+            null,
+            page
+        ).enqueue(object :
+            Callback<ResponseMyAnswer> {
+            override fun onResponse(
+                call: Call<ResponseMyAnswer>,
+                response: Response<ResponseMyAnswer>
+            ) {
+                if (response.isSuccessful) {
+                    copyMyAnswerList = response.body()!!.data?.answers?.toMutableList()
+                    _mypageWriteData.value=copyMyAnswerList.toMutableList()
```

Comment:
굳이 꼬아서 갈 필요가 없을 것 같아. 복사는 한 번이면 충분해. 그 이상의 변수 사용은 불필요하고 코드가 복잡하게 보여
```suggestion
                    _mypageWriteData.value=response.body()!!.data?.answers?.toMutableList()
```